.github/workflows/node-test.yml

```diff
@@ -8,6 +8,28 @@ env:
   CI: true
 
 jobs:
+  lint:
+    runs-on: ubuntu-latest
+    if: github.event_name == 'push'
```

Comment:
I think we want this to be pull_request so that 3p contribs hit it

Reply:
done.